launcher/game/preferences.rpy

```diff
@@ -334,19 +332,19 @@ screen preferences():
 
                             add HALF_SPACER
 
-                            textbutton _("Orphan translations"):
+                            textbutton _("Check for orphan/obsolete translations"):
                                 style "l_checkbox"
-                                action ToggleSetMembership(persistent.lint_options, "--orphan-tl")
-                            textbutton _("Parameters overriding builtin names"):
+                                action InvertSelected(ToggleSetMembership(persistent.lint_options, "--no-orphan-tl"))
+                            textbutton _("Check for parameters overriding builtin names"):
```

Comment:
"Shadowing" is the correct term for this, overriding is something different, and it would be the builtins themselves, not their names in either case.